src/email/send.controller.spec.js

```diff
@@ -0,0 +1,26 @@
+describe('Email Send Controller', function() {
+    var scope, controller, $q, emailSendService;
+
+    beforeEach(module('motech-email'));
+    beforeEach(inject(function($rootScope, $controller, _$q_, EmailSendService){
+        scope = $rootScope.$new();
+        controller = $controller('EmailSendController', {
+            $scope: scope,
+        });
+        $q = _$q_;
+        emailSendService = EmailSendService;
+    }));
+
+    it('should execute EmailSendService.send function', function() {
+        scope.mail = {
+            fromAddress : "soldevelo@soldevelo.com",
```

Comment:
@DamianNowakSoldevelo Change fromAddress to something more universal in Motech, like: motech@motech.com